subjects/cameltosnakecase/README.md

```diff
@@ -0,0 +1,54 @@
+## Camel-to-snake-case 
+
+### Instructions
+Write a function that converts a string from `CamelCase` to `snake_case`.
+
+Camel case is the practice of writing phrases without spaces or punctuation, it indicates the separation of words with a single capitalized letter. Snake case is a style of writing in which each space is replaced by an underscore (_) character.
+
+Here are some rules for you to follow:
+
+- If the string is empty, return an empty string.
+- If the string is not `CamelCase`, return the string unchanged.
+- If the string is `CamelCase`, return the `snake_case` version of the string.
+
+Basic `CamelCase` Capitalization Rules:
+
+- The first letter must be capitalized.
+- The word must not have two capitalized letters together (CamelCAse) not end with a capitalized letter (CamelCasE).
```

Comment:
instead of the second "not" leave the "nor" as ti was it looks better.

"(CamelCAse) nor end with a capitalized letter (CamelCasE)."